spec.bs

```diff
@@ -2444,6 +2448,28 @@ algorithms to achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-notification.https.html
 </wpt>
 
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
+
+The [[!CSSOM-VIEW]] specification defines the {{Element/scrollIntoView()}} method that calls the
+[=scroll a target into view=] algorithm. This will not only scroll the {{Element}} or
+[=css2/viewport=] to make the target visible, but will also scroll [=tree/ancestor=]s if necessary
+to make the target visible, essentially causing the scroll to "bubble up". This means that
+{{Element/scrollIntoView()}} performed in a [=child navigable=] or [=fenced navigable
+container/fenced navigable=] can be observed by its embedder, allowing for collusion across a fenced
+frame boundary. This section patches {{Element/scrollIntoView()}} to prevent that collusion at the
```

Comment:
We're actually patching the "scroll a target into view" algorithm specifically, so let's mention that _that's_ what we're patching.